src/components/app.jsx

```diff
@@ -10,26 +10,26 @@ class App extends React.Component {
         this.handleSubmit = this.handleSubmit.bind(this);
     }
 
-    async handleSubmit(cityID) {
+    async handleSubmit(cityId) {
         try {
-            console.log(await this.getForecasts(cityID));
+            console.log(await this.getForecast(cityId));
         } catch (err) {
             console.error(err);
         }
         
     }
 
-    async getForecasts(cityID) {
+    async getForecast(cityId) {
         try {
             const fiveDayForecast = await axios.get('http://api.openweathermap.org/data/2.5/forecast', {
                 params: {
-                    id: cityID,
+                    id: cityId,
                     APPID: process.env.OPENWEATHER_API_KEY,
                     units: 'metric'
                 }
             });
             
-            const threeDayForecastList = fiveDayForecast.data.list.slice(0, 21);
+            const threeDayForecastList = fiveDayForecast.data.list.slice(0, 25);
```

Comment:
Make sure that date-time is in Ukrainian local time and adjust number of cards to not exceed three days